airbyte-ci/connectors/connectors_insights/src/connectors_insights/consts.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+DOCKER_INDEX = "docker.io"
+DOCKERFILE_NAME = "Dockerfile"
+LOW_CODE_MANIFEST_FILE_NAME = "manifest.yaml"
+METADATA_FILE_NAME = "metadata.yaml"
+POETRY_LOCK_FILE_NAME = "poetry.lock"
+PYPROJECT_FILE_NAME = "pyproject.toml"
+SETUP_PY_FILE_NAME = "setup.py"
```

Comment:
Reminder to track the idea to rip out everything related to setup.py after we fully migrate.